src/providers/LedgerProvider.js

```diff
@@ -23,33 +23,35 @@ export default class LedgerProvider extends WalletProvider {
   }
 
   async createTransport () {
-    if (!LedgerProvider.transport) {
+    if (!this._transport) {
       debug('creating ledger transport')
-      LedgerProvider.transport = await Transport.create()
+      this._transport = await Transport.create()
       debug('ledger transport created')
-      LedgerProvider.transport.on('disconnect', () => {
+      this._transport.on('disconnect', () => {
         debug('ledger disconnected')
         this._appInstance = null
-        LedgerProvider.transport = null
+        this._transport = null
```

Comment:
Earlier we've used static property because the transport is independent of ledger apps. Can you confirm that this change is necessary?

Reply:
In my tests I found out the once the transport is initiated with one app we don't manage to reuse the same with another app